lib/puppet/defaults.rb

```diff
@@ -590,9 +590,11 @@ def self.initialize_default_settings!(settings)
     },
     :trusted_external_command => {
       :default  => nil,
-      :desc     => "The external trusted facts script to use.
+      :type     => :file_or_directory,
+      :desc     => "The external trusted facts script or directory to use.
```

Comment:
Is `desc` done separately or are my changes OK here? Thinking I should mention that if it's a dir, we store the facts in `trusted['external'][<basename>]`...